src/cmd/team.go

```diff
@@ -231,14 +361,18 @@ EOF
 
 func init() {
 	createCmd.AddCommand(createTeamCmd)
-	createCmd.AddCommand(createMemberCmd)
-	createCmd.AddCommand(createContactCmd)
+	createTeamCmd.AddCommand(createMemberCmd)
```

Comment:
I changed some of these since I think they were incorrect and causing an issue. For example, if I would try to create a team member doing something like `create team member Z2lkOi8vb3BzbGV2ZWwvVGVhbS8xMTQ 
 "jackie@opslevel.com"` I would get this error
```
Error: accepts 1 arg(s), received 3
Usage:
  opslevel create team NAME [flags]
```
It thinks I'm trying to create a team; same issue for creating team contacts and team tags. I noticed in `src/cmd/service.go` that we called `AddCommand` on the original create/get/delete command for the object instead of on `createCmd `, and when I updated this file to follow a similar pattern it fixed this issue.

Let me know if this is incorrect and I can revert this change

Reply:
discussed this, and turned out to just be a misunderstanding about how to run this command